install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -1,10 +1,11 @@
 dependencies:
   - name: pilot
     version: 0.3.0
-    condition: istio.pilot_enabled
+    condition: global.pilot.enabled   
```

Comment:
Look like there's extra whitespace at the end of the line.  I'm not sure how picky istio (either the culture or the product) is about such things.  This file and others cite 0.3.0 or 0.4.0 but I'm not sure if these are stale references that should be updated via updateVersion.sh.

Reply:
Thanks, I'll correct it